node/cli/src/cli.rs

```diff
@@ -107,6 +109,12 @@ pub struct Cli {
 	pub relaychain_args: Vec<String>,
 }
 
+impl Cli {
+	pub fn node_name() -> String {
+		env::var(NODE_NAME_ENV).unwrap_or("Unknown".into())
```

Comment:
the logic must be
1) if there env than use it else default
2) Default "node name" must be constructed through features:
if feature unique-runtime than "Unique"
else if feature quartz-runtime than "Quartz"
else "Opal"